### PR TITLE
test: added test case to verify data among pages based on project change and refactoring of other tests

### PR DESCRIPTION
**Pull request by carpe-github**
in scope of this PR:

1. added test case to verify data among pages based on project change - page-data-update-project-switch.spec.ts.
2. made improvements in support/commands.ts
3. improved bridge/cypress/integration/uniform-integrations.spec.ts to store locators in page object classes as constants

### CODE REVIEW

File: bridge/cypress/integration/page-data-update-project-switch.spec.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    const settingsPage = new SettingsPage();
+    const PROJECT_DYNATRACE = 'dynatrace';
+    const PROJECT_TEST = 'testproject';
+    const QG_DYNATRACE = 'quality-gate';
+    const QG_TEST_PROJ = 'newproj-quality-gate';
+    const SERVICE_DYNATRACE = 'items';
+    const SERVICE_TEST_PROJ = 'newService';
+    const UNIFORM_REMEDIATION = 'remediation-service';
+    const UNIFORM_APPROVAL_SERVICE = 'approval-service';
+    const UNIOFORM_DYNATRACE_SERVICE = 'dynatrace-service';
```
Comment (laneli) [inline bridge/cypress/integration/page-data-update-project-switch.spec.ts:23]: Typo ...
`UNIOFORM_DYNATRACE_SERVICE` > `UNIFORM_DYNATRACE_SERVICE`

File: bridge/cypress/integration/page-data-update-project-switch.spec.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    basePage.goToServicesPage();
+    cy.get(servicePage.SERVICE_PANEL_TEXT_LOC).contains(SERVICE_TEST_PROJ);
+
+    basePage.chooseProjectFromHeaderMenu(PROJECT_DYNATRACE);
+    cy.get(servicePage.SERVICE_PANEL_TEXT_LOC).contains(SERVICE_DYNATRACE);
+
+    basePage.goToUniformPage();
+    cy.get(uniformPage.UNIFORM_NAME_LOC).contains(UNIFORM_APPROVAL_SERVICE);
+    cy.get(uniformPage.UNIFORM_NAME_LOC).contains(UNIFORM_REMEDIATION);
+    cy.get(uniformPage.UNIFORM_NAME_LOC).contains(UNIOFORM_DYNATRACE_SERVICE);
```
Comment (laneli) [inline bridge/cypress/integration/page-data-update-project-switch.spec.ts:70]: typo

File: bridge/cypress/integration/page-data-update-project-switch.spec.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+    basePage.chooseProjectFromHeaderMenu(PROJECT_TEST);
+    cy.get(uniformPage.UNIFORM_NAME_LOC).contains(UNIFORM_APPROVAL_SERVICE);
+    cy.get(uniformPage.UNIFORM_NAME_LOC).contains(UNIFORM_REMEDIATION);
+    cy.get(uniformPage.UNIFORM_NAME_LOC).contains(UNIOFORM_DYNATRACE_SERVICE);
```
Comment (laneli) [inline bridge/cypress/integration/page-data-update-project-switch.spec.ts:75]: typo

**Review by laneli — CHANGES_REQUESTED** [review]

File: bridge/cypress/integration/page-data-update-project-switch.spec.ts (hunk 2 above)
Comment (carpe-github) [reply bridge/cypress/integration/page-data-update-project-switch.spec.ts]: Thanks for the finding, fixed!

File: bridge/cypress/integration/page-data-update-project-switch.spec.ts (hunk 3 above)
Comment (carpe-github) [reply bridge/cypress/integration/page-data-update-project-switch.spec.ts]: Thanks for the finding, fixed!

File: bridge/cypress/integration/page-data-update-project-switch.spec.ts (hunk 1 above)
Comment (carpe-github) [reply bridge/cypress/integration/page-data-update-project-switch.spec.ts]: Thanks for the finding, fixed!

**Review by laneli — APPROVED** [review]

**Review by laneli — APPROVED** [review]

**Pull request merged**